Region/CrtfImportExport.cc

```diff
@@ -336,14 +336,19 @@ void CrtfImportExport::ImportAnnSymbol(casacore::CountedPtr<const casa::Annotati
         point.set_y(pixel_coords[1]);
         control_points.push_back(point);
 
-        // Other RegionState params
-        std::string name = annotation_region->getLabel();
+        // Set other RegionState parameters
         CARTA::RegionType type(CARTA::RegionType::POINT);
         float rotation(0.0);
+        RegionState region_state(_file_id, type, control_points, rotation);
+
+        // Set RegionStyle
+        RegionStyle region_style = ImportStyleParameters(annotation_region);
 
-        // Create RegionState
-        RegionState region_state = RegionState(_file_id, name, type, control_points, rotation);
-        _import_regions.push_back(region_state);
+        // Add RegionProperties to list
+        RegionProperties region_properties;
+        region_properties.state = region_state;
+        region_properties.style = region_style;
+        _import_regions.push_back(region_properties);
```

Comment:
RegionState and RegionStyle are kept separated because that is how they are returned to the frontend in the ack message.  Perhaps I should use CARTA::RegionInfo and CARTA::RegionStyle instead so I do not have to convert the struct to a submessage.

Reply:
I see, the RegionProperties constructor could create the RegionInfo and RegionStyle.

Reply:
Yup. In the initializer list. Then you could collapse everything into one line with `_import_regions.push_back(RegionProperties(blah...));`, although maybe that's a bit *too* terse and would be better as two lines.

Reply:
Clearly I did not refactor adequately when style parameters were removed from RegionInfo.  Now all Import*Region methods in CrtfImportExport and Ds9ImportExport return the RegionState, then common code (not repeated) to call ImportStyleParameters to return RegionStyle, then construct RegionProperties(RegionState&, RegionStyle&).  I left these as structs since RegionState is passed to the Region constructor, and it's more consistent and easier to set the RegionStyle struct than the CARTA::RegionStyle message.